app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -72,7 +69,6 @@ def update
     api :POST, '/organizations', N_('Create organization')
     param :name, String, :desc => N_("name"), :required => true
     param :label, String, :desc => N_("unique label")
-    param :description, String, :desc => N_("description")
```

Comment:
@daviddavis if the error doesn't appear for you it might be something I didn't setup correctly, can you see description in the apidocs after my change?